server/chat-commands/core.ts

```diff
@@ -1450,7 +1461,13 @@ export const commands: Chat.ChatCommands = {
 	 * Challenging and searching commands
 	 *********************************************************/
 
+	'search': 'battle',
+	battle!: 'search',
 	async search(target, room, user, connection) {
+		if (cmd === 'battle') {
+			return this.sendReply(this.tr`What?! How are you not more excited to battle?! Try /battle! to show me you're ready.`);
```

Comment:
Do we really need this joke? Typing ``battle!`` can be non-trivial for some people. E.g. In Spanish, ``!`` may be auto-corrected to ``¡`` (opening exclamation mark), which can turn into an accessibility nightmare.

Reply:
(For total clarity, I did not create the joke, it was already there. I'm pretty neutral on keeping it or not.)

Reply:
Yes, I am aware. I'm just raising it, since this subject can probably be discussed too in this PR.

Reply:
Yeah I'm fine getting rid of it.